snap/snapcraft.yaml

```diff
@@ -571,7 +626,7 @@ parts:
         rm pbm2l7k.xml; \
         rm pbm2lwxl.xml; \
         rm pentaxpj.xml; \
-      	rm ppmtomd.xml; \
+      rm ppmtomd.xml; \
       )
```

Comment:
For what is this change good for? If it is not intended, please revert it, to avoid clutter.